operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -109,6 +109,12 @@ type CentralComponentSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=4
 	Persistence *Persistence `json:"persistence,omitempty"`
 
+	// Enable necessary components for provisioning of monitoring metrics for
+	// Central and other components of ACS stack. This is required if metrics
+	// are collected by Observability Operator.
```

Comment:
Since we also will have this in upstream operator, does it make sense to mention the `Observability operator` here? 
From the customer's perspective, this will expose the prometheus port, using the observability operator is something specific to managed services, I'd say we are better off removing this and mentioning that it will expose a metrics port at 9090? Same applies to all occurrences.

Reply:
@dhaus67 @stehessel I have changed the structure a bit, based on the comment from @porridge. Now the structure is `central.monitoring.exposeEndpoint` - with string value `Enabled|Disabled`.

Please check the descriptions in the comments again. Only comments are relevant, everything else is automatically generated.